### PR TITLE
Update docs and helptext

### PR DESCRIPTION
**Pull request by Billy99**
Perform a scrub of bpfctl helptext and after the recent changes

Resolves: #725

### CODE REVIEW

File: bpfctl/src/main.rs [hunk 1]
```diff
@@ -69,11 +69,15 @@ struct ListArgs {
     #[clap(short, long, verbatim_doc_comment, hide_possible_values = true)]
     program_type: Option<ProgramType>,
 
-    // Optional: List programs which contain a specific set of metadata labels
+    /// Optional: List programs which contain a specific set of metadata labels
+    /// that were applied when program was loaded with `--metadata` parameter.
```
Comment (astoycos) [inline bpfctl/src/main.rs:73]:
```suggestion
    /// that were applied when the program was loaded with `--metadata` parameter.
```

File: bpfctl/src/main.rs [hunk 2]
```diff
@@ -85,7 +89,7 @@ struct LoadFileArgs {
     #[clap(short, long, verbatim_doc_comment)]
     path: String,
 
-    /// Required: Name of the ELF section from the object file.
+    /// Required: The name of the function that is the entry point for the BPF program..
```
Comment (astoycos) [inline bpfctl/src/main.rs:92]:
```suggestion
    /// Required: The name of the function that is the entry point for the BPF program.
```

File: bpfctl/src/main.rs (hunk 1 above)
Comment (Billy99) [reply bpfctl/src/main.rs]: done

File: bpfctl/src/main.rs (hunk 2 above)
Comment (Billy99) [reply bpfctl/src/main.rs]: Done

**Review by astoycos — APPROVED** [review]
LGTM

Thanks @Billy99!

**Pull request merged**